src/components/screens/monitor/delegates/forgingDetails.js

```diff
@@ -22,8 +24,11 @@ const Forger = ({ forger }) => (
 );
 
 const ForgingDetails = ({
-  t,
+  t, networkStatus,
 }) => {
+  const initialSupply = 10000000000000000;
```

Comment:
This might make the value misleading when used with a different blockchain than mainnet. I've checked the Core API and there is no endpoint telling us what was the initial supply.

Consider making it hardcoded, based on NetworkId/nethash.

Reply:
I thought the initial supply is the same for all networks.

Reply:
The testnet supply is 139,576,098 and the mainnet supply is 139,630,112. The only difference according to Manu is 
> mainnet is burning a lot faster and consistent than testnet.